src/components/DescriptionUrls.vue

```diff
@@ -12,16 +12,34 @@ export default {
   name: 'DescriptionUrls',
   props: {
     description: {
-      type: Array,
+      type: String,
       required: true
     }
   },
+  computed: {
+    descriptionSplit () {
+      return this.textURLSplit(this.description)
+    }
+  },
   methods: {
+    // Returns an array of structure [{text, url}, {text, url}, ...]
+    // Note: split may be [{text}] if no url is found
+    textURLSplit (text) {
+      const split = text.split(/(?:[^\S]|^)((?:(?:https?:\/\/)|(?:www\.))(?:\S+))/gi)
+      return split.reduce((accumulator, current, index) => {
+        if (index % 2) {
+          return accumulator
+        } else {
+          return [...accumulator, { text: current, url: split[index + 1] }]
+        }
+      }, [])
+    },
     ExternalURL (url) {
       if (!url.match(/^[a-zA-Z]+:\/\//)) {
-        url = 'https://' + url
+        return 'https://' + url
```

Comment:
what does this actually do? only check for protocol? then you could better check this regex: /^http/gmi